src/renderer/modals/Swap/SwapBody.js

```diff
@@ -81,6 +82,7 @@ const SwapBody = ({
       title={<Trans i18nKey="swap.modal.title" />}
       render={() => (
         <>
+          <TrackPage category="Swap" name={`ModalStep${activeStep + 1}`} />
```

Comment:
It's better to use the name of the step in case order changes in future

Reply:
Sure, makes sense

Reply:
In fact, `activeStep` is not even a number but rather the id already, so this would've done something like `ModalStepfinished1` which makes no sense. Fixed